Marketplace/Getting Started Guides/getting-started-alert-logic-log-manager.md

```diff
@@ -0,0 +1,80 @@
+{{{
+"title": "Getting Started Guide: Alert Logic Log Manager",
+"date": "04-05-2017",
+"author": "",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+![Alert Logic-logo](../../images/Marketplace/Alert-Logic-logo.png)
+
+### Summary
+
+Alert Logic Log Manager with ActiveWatch is a Security-as-a-Service (SaaS) solution that meets compliance requirements and identifies security issues across your entire environment, including public cloud. It collects, processes and analyzes your data, while our ActiveWatch team unlocks the insights hidden in your log data with 24x7 expert monitoring and analysis.
+
+Alert Logic Log Manager collects, aggregates and normalizes log data from your CenturyLink Cloud servers. The web interface provides a unified view into all of your data, with tools to rapidly uncover the insight and alerts you need to remain secure and compliant.
+
+### Description
+
+Alert Logic Log Manager provides increased visibility, rapid custom reporting, and scalable, real-time log collection and log management for a predictable monthly fee. Our cloud security solution:
+
+* Deploys rapidly, wherever your IT resides
+
+* Scales to your particular business needs
+
+* Provides deep security insight into your environment with an intuitive interface
+
+* Collects syslog from firewalls, switches, routers, printers, Unix servers and many more devices
+
+* Turns your data into action, delivering correlations and validated incidents directly to you
+
+* This diagram illustrates the Log Manager virtual appliance architecture.
+
+![alertlogic-diagram](../../images/Marketplace/alertlogic-logmgr-appliance-architecture.png)
+
+### Additional Information
+
+For more details on the Alert Logic Log Manager architecture, please visit the [Log Manager Documentation page](https://docs.alertlogic.com/products101/log-manager-101.htm).
+
+### Deployment Process
+
+This Runner job performs the following steps:
+
+1. Provisions an Alert Logic Log Manager virtual appliance in the customer's CenturyLink Cloud account and initiates a monthly recurring subscription based on the selected license (e.g. Support for up to 5, 10, 15, 20 or more server "nodes").
```

Comment:
looking at runner there is no license qty button.  is this step valid? or just not live yet?